source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -284,18 +294,25 @@ void RedisCluster::RedisDiscoverySession::onResponse(
     }
 
     // Field 2: Master address for slot range
-    // TODO(hyang): For now we're only adding the master node for each slot. When we're ready to
-    //  send requests to replica nodes, we need to add subsequent address in the response as
-    //  replica nodes.
     auto master_address = ProcessCluster(slot_range[2]);
     if (!master_address) {
       onUnexpectedResponse(value);
       return;
     }
-    slots_.emplace_back(slot_range[0].asInteger(), slot_range[1].asInteger(), master_address);
+
+    slots_->emplace_back(slot_range[0].asInteger(), slot_range[1].asInteger(), master_address);
+
+    for (auto slave = std::next(slot_range.begin(), 3); slave != slot_range.end(); ++slave) {
```

Comment:
Can you move the magic integers into a local constant with a name? Here and above.